docs/6.x/cluster.md

```diff
@@ -1437,6 +1438,32 @@ The CoreDNS configuration can be edited after installation, by updating the `kub
 [RBAC]: https://kubernetes.io/docs/admin/authorization/rbac/
 [promiscuous-mode]: https://en.wikipedia.org/wiki/Promiscuous_mode
 
+## Interacting with Cluster Registry
+
+Gravity clusters run private Docker registries on the master nodes. These registries are maintained
+in-sync by Gravity and contain Docker images for Gravity cluster and application images.
+
+To view a list of Docker images currently published in the cluster registry, Gravity provides
+a convenience command:
+
+```bash
+$ gravity registry list
+```
+
+By default the command will contact the registry server running at `registry.local:5000` address
+which resolves to the registry running on the currently active master node. To list images in a
+specific registry, provide a `--registry` flag to the command:
+
+```bash
+$ gravity registry list --registry=192.168.1.1:5000
```

Comment:
Should we make it possible to use just the hostname/IP address (w/o the port) as this part is fixed?

Reply:
Yeah, good point. Could default to 5000 if not specified.